parachain/pallets/basic-channel/src/inbound/benchmarking.rs

```diff
@@ -0,0 +1,255 @@
+//! BasicInboundChannel pallet benchmarking
+
+#![cfg(feature = "runtime-benchmarks")]
```

Comment:
Can remove this if we only declare the benchmarking module if the feature is enabled.

Reply:
I agree with not declaring the module if it's not used. I'm going to leave it as is in this PR though, and update all our pallets in another PR. We're using this same pattern in all pallets, and I'd rather update them all in one go to keep the style consistent.